packages/communication-react/review/communication-react.api.md

```diff
@@ -325,8 +325,15 @@ export type CallCompositeProps = {
     fluentTheme?: PartialTheme | Theme;
     callInvitationURL?: string;
     onRenderAvatar?: (props: PlaceholderProps, defaultOnRender: (props: PlaceholderProps) => JSX.Element) => JSX.Element;
+    overrideCallControlButtons?: (defaultButtons: CallControlButtonCollection) => CallControlButtonCollection;
```

Comment:
Mixing string literal types with actual objects in a single union type makes me squirm a little.

Technically, Contoso could ignore all the `defaultButtons` provided here, and build out all the button themselves from the button components. So we don't gain any safety by hiding the constructed buttons (and only providing names to this callback).

How about making `CallControlButtonCollection` only contain actual buttons. And the callback gets the fully constructed `defaultButtons` that Contoso is expected to pass back to us along with their own buttons?